java/core/src/main/java/co/worklytics/psoxy/rules/RulesUtils.java

```diff
@@ -117,6 +129,44 @@ public List<StorageHandler.ObjectTransform> parseAdditionalTransforms(ConfigServ
         }
     }
 
+    public Optional<String> getDefaultScopeIdFromRules(com.avaulta.gateway.rules.RuleSet ruleSet) {
+        if (ruleSet instanceof RuleSet) {
+            return Optional.ofNullable(((RuleSet) ruleSet).getDefaultScopeIdForSource());
```

Comment:
if think if `null`, then`instance Ruleset` is `false` right? Although it shouldn't be `null`

Reply:
you are right, so this can be straight, the casting is not needed
```suggestion
            return Optional.of(ruleSet.getDefaultScopeIdForSource());
```